<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>theWeb</string>
  </property>
  <property name="windowIcon">
   <iconset theme="theweb" resource="resources.qrc">
    <normaloff>:/icons/theweb.svg</normaloff>:/icons/theweb.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="tabFrame" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="tabLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="window-close"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="tabs"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewTab"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionGoBack"/>
    <addaction name="actionGoForward"/>
    <addaction name="actionReload"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGoBack"/>
   <addaction name="actionGoForward"/>
   <addaction name="actionReload"/>
   <addaction name="actionNewTab"/>
  </widget>
  <action name="actionGoBack">
   <property name="icon">
    <iconset theme="go-previous"/>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionGoForward">
   <property name="icon">
    <iconset theme="go-next"/>
   </property>
   <property name="text">
    <string>Go Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="icon">
    <iconset theme="tab-new"/>
   </property>
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
